src/lib/users/widgets/public_key_selector.rb

```diff
@@ -234,20 +242,25 @@ def update
       #
       # @return [String] Key content
       def read_key
-        select_blk_device
-        dir = Dir.mktmpdir
-        begin
-          mounted = Yast::SCR.Execute(
-            Yast::Path.new(".target.mount"), [selected_blk_device_name, dir], "-o ro"
-          )
-          if mounted
-            read_key_from(dir)
-          else
-            report_mount_error(selected_blk_device_name)
+        if Yast::Arch.is_wsl
+          # /mnt has windows drives mounted by default
+          read_key_from("/mnt")
```

Comment:
It's better to use `Yast::Installation.destdir` instead of the hardcoded `/mnt`. (You might need to add `Yast.import "Installation"`.)

Reply:
it's unrelated to destdir of an installation actually. MS decided to use /mnt for mounting Windows drives. See eg https://docs.microsoft.com/de-de/windows/wsl/faq